readme.md

```diff
@@ -0,0 +1,40 @@
+# City Temperature Management API
+This project is built on FastAPI and provides functionalities related to managing cities and temperature records.
+
+To clone this project from GitHub, follow these steps:
+1. Open your terminal or command prompt.
+2. Navigate to the directory where you want to clone the project.
+3. Run the following command:
+```shell
+git clone https://github.com/YakovK1808/py-fastapi-city-temperature-management-api.git
+cd py-fastapi-city-temperature-management-api
+python -m venv venv
+venv\Scripts\activate  #for MacOS/Linux use: source vevn/bin/activate
+```
+4. Install requirements:
+```shell
+pip install -r requirements.txt
+```
+5. You should register on weatherapi.com and get your WEATHER_API_KEY
+6. Copy .env.sample and create .env file. After that you need to put your own WEATHER_API_KEY instead of default one.
+## Adding Migrations
+This project uses Alembic for managing database migrations. Execute the following commands to create and run migration:
+```shell
+alembic revision --autogenerate -m "initial_migrations"
+alembic upgrade head
+```
+## Running the Server
+To run the server execute the following command:
+hypercorn main:app --reload

```

Comment:
![image](https://github.com/mate-academy/py-fastapi-city-temperature-management-api/assets/87720583/6c2ae0a8-b547-4358-a4d2-9d74af586ad1)
Wiil be good to format by Markdown